system/Validation/FileRules.php

```diff
@@ -122,6 +122,11 @@ public function max_size(string $blank = null, string $params, array $data): boo
 
 		$file = $this->request->getFile($name);
 
+		if($file->getError() === UPLOAD_ERR_NO_FILE)
```

Comment:
Shouldn't this test follow the is_null($file) block?
The order you have it breaks travis-ci, and it looks like rightfully so :(